next.config.js

```diff
@@ -0,0 +1,45 @@
+const withLess = require('@zeit/next-less');
+const withCss = require('@zeit/next-css');
+const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === 'true',
+});
+
+module.exports = withBundleAnalyzer(
+  withCss(
+    withLess({
+      cssModules: true,
+      webpack: (config, options) => {
+        if (config.resolve.plugins) {
+          config.resolve.plugins.push(new TsconfigPathsPlugin());
+        } else {
+          config.resolve.plugins = [new TsconfigPathsPlugin()];
+        }
+        config.module.rules.push({
+          test: /\.(eot|woff|woff2|ttf|svg|png|jpg|jpeg|gif)$/,
+          use: {
+            loader: 'url-loader',
+            options: {
+              limit: 5000,
```

Comment:
It is a limit on the file size. If I remember my webpack configurations correctly it is used to determine if you inline an image as a base64 string directly in the js code instead of a URL, or if it is copied to the public directory and a URL is placed in the image source.

If the image is small enough is will be much faster and efficient to include the base64 encoded 'data-url', as you don't need a separate request to fetch the image. But if your images is too large it may have a massive impact on the bundle size and initial load times of the app